docs/tutorial/6-connect-a-terminal.md

```diff
@@ -13,7 +13,7 @@ Your Express app is up and running and the preview window updates automatically
 
 ## 1. Install `Xterm.js`
 
-The output of the processes will be printed to a `div` and the messages will feature different colors or styles. In order to achieve this, we will use [`Xterm.js`](https://xtermjs.org), which is a terminal frontend component. To install it, run the following command in your development terminal:
+The terminal frontend component that we will use is [`Xterm.js`](https://xtermjs.org). It's the same terminal that is used by Visual Studio Code and many other web-based IDEs. To install it, run the following command in your development terminal:
```

Comment:
Not sure about this change. Feel free to reiterate.

Reply:
It’s great!